python-template/{{cookiecutter.placeholder_repo_name}}/README.rst

```diff
@@ -2,32 +2,27 @@
 #############################
 
 |pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
-|license-badge|
+|license-badge| |status-badge|
 
-{{ cookiecutter.project_short_description}}
-
-The ``README.rst`` file should start with a brief description of the repository,
-which sets it in the context of other repositories under the ``edx``
-organization. It should make clear where this fits in to the overall edX
-codebase.
+Purpose
+*******
 
-Overview (please modify)
```

Comment:
I wonder if there should be some replacement for "(please modify)". Could be something like:
```
.. note::

  This README was auto-generated. The "Purpose" needs to be manually updated, and the rest of the README should be reviewed for potential updates.
```

Reply:
@feanil: Can you confirm your intention? Not sure if you intentionally don't want to add a note, or if you just missed this. Thanks.

Also note that if you read the README, it is clear that it should be updated. But, I'm just wondering if anything can make it more clear for cases where your eyes are passing over it, but you aren't actually reading it. It might be that mu suggestion is no better than what is there.

Reply:
Missed this the first time, I'll add it along with the other suggestions.